addons/website_quote/controllers/main.py

```diff
@@ -18,50 +15,53 @@ class sale_quote(http.Controller):
     def view(self, order_id, pdf=None, token=None, message=False, **post):
```

Comment:
order to params is confusing, better to have order_id, token=None, pdf=None, message=None, ....